src/app/core/models/github/github-graphql.commitstats.model.ts

```diff
@@ -0,0 +1,14 @@
+import { CommitStatsFragment } from '../../../../../graphql/graphql-types';
+import { GithubCommit } from './github-commit.model';
+
+export class GithubGraphqlCommitStat extends GithubCommit {
+  constructor(stats: CommitStatsFragment) {
+    super();
+    this.additions = stats.additions;
+    this.deletions = stats.deletions;
+    this.setDate(stats.committedDate);
+    this.url = stats.url;
+    this.messageHeadline = stats.messageHeadline;
+    this.messageBody = stats.messageBody;
+  }
+}
```

Comment:
Only concern is if we want to use readonly in GithubCommit so that variables in GithubCommit cannot be changed. But that can be for future refactoring, I think this is ok for now